bin/bazel_to_go.py

```diff
@@ -223,14 +223,21 @@ def template_protos(WKSPC):
                         # check if there are files under /template/<some uber dir>/<some template dir>
                         if file2.endswith("_tmpl.pb.go") or file2.endswith("handler.gen.go"):
                             makelink(WKSPC + "/bazel-genfiles/template/" + template + "/" + file + "/" + file2, WKSPC + "/template/" + template + "/" + file + "/" + file2)
-    for template in os.listdir(WKSPC + "/bazel-genfiles/test/template"):
+
+    for d in os.listdir(WKSPC + "/bazel-genfiles/test/e2e"):
+        if os.path.isdir(WKSPC + "/bazel-genfiles/test/e2e/" + d):
+            for file in os.listdir(WKSPC + "/bazel-genfiles/test/e2e/" + d + "/spyAdapter"):
+                if file.endswith(".gen.go"):
+                    makelink(WKSPC + "/bazel-genfiles/test/e2e/" + d + "/spyAdapter/" + file, WKSPC + "/test/e2e/" + d + "/spyAdapter/" + file)
+
+    for template in os.listdir(WKSPC + "/bazel-genfiles/test/e2e/advance/template"):
         if template.endswith(".gen.go"):
-            makelink(WKSPC + "/bazel-genfiles/test/template/" + template, WKSPC + "/test/template/" + template)
-        if os.path.isdir(WKSPC + "/bazel-genfiles/test/template/" + template):
-            for file in os.listdir(WKSPC + "/bazel-genfiles/test/template/" + template):
+            makelink(WKSPC + "/bazel-genfiles/test/e2e/advance/template/" + template, WKSPC + "/test/e2e/advance/template/" + template)
```

Comment:
change makelink interface to take multiple strings makelink(dir, *path)
and implement string concat there once.